data_loader/training_extractor.py

```diff
@@ -12,20 +12,53 @@
 
 # The ID and range of a sample spreadsheet.
 SAMPLE_SPREADSHEET_ID = '1NGRUyzImlaUd-UrTkNXSd1JB7OOHnw6e1h4AohRXFK8'
+TEST_INSERT_SPREADSHEET_ID = '1KQ2nFO9oeDLhHmKh9UTvNnhepKhMuTOyKmh7Nnxbj80'
```

Comment:
That should be in a test file that tests your logic.
(we don't mix production code, and test code in the same file)